addons/im_livechat/static/tests/helpers/core/web/setup_manager_patch.js

```diff
@@ -0,0 +1,16 @@
+/* @odoo-module */
+
+import { livechatCoreWeb } from "@im_livechat/core/web/livechat_core_web_service";
+
+import { setupManager } from "@mail/../tests/helpers/webclient_setup";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(setupManager, "im_livechat/core/web", {
+    setupServices(...args) {
+        return {
+            ...this._super(...args),
+            "im_livechat.core.web": livechatCoreWeb,
+        };
+    },
+});
```

Comment:
I think there's an ongoing PR (or even already merged) that make it unnecessary to patch `setupServices` anymore? Might drop these if it's merged!

Reply:
Yes I cant' wait for it, I requested it because it was becoming too cumbersome to manage with this.